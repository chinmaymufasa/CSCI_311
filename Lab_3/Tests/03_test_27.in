2
161
5
2
100
5
2
105
5
2
115
5
2
113
5
2
67
5
2
199
5
2
36
5
2
21
5
2
7
5
2
4
5
2
16
5
2
57
5
2
173
5
2
17
5
2
178
5
2
190
5
2
172
5
2
119
5
2
51
5
1
20
-1