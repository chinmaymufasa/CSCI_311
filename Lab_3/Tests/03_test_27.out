161 
100 161 
100 105 161 
100 105 115 161 
100 105 113 115 161 
67 100 105 113 115 161 
67 100 105 113 115 161 199 
36 67 100 105 113 115 161 199 
21 36 67 100 105 113 115 161 199 
7 21 36 67 100 105 113 115 161 199 
4 7 21 36 67 100 105 113 115 161 199 
4 7 16 21 36 67 100 105 113 115 161 199 
4 7 16 21 36 57 67 100 105 113 115 161 199 
4 7 16 21 36 57 67 100 105 113 115 161 173 199 
4 7 16 17 21 36 57 67 100 105 113 115 161 173 199 
4 7 16 17 21 36 57 67 100 105 113 115 161 173 178 199 
4 7 16 17 21 36 57 67 100 105 113 115 161 173 178 190 199 
4 7 16 17 21 36 57 67 100 105 113 115 161 172 173 178 190 199 
4 7 16 17 21 36 57 67 100 105 113 115 119 161 172 173 178 190 199 
4 7 16 17 21 36 51 57 67 100 105 113 115 119 161 172 173 178 190 199 
Not found
